arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/CallSection.java

```diff
@@ -163,8 +163,10 @@ public CallSection(Hub hub, MessageFrame frame) {
 
     if (Exceptions.any(exceptions)) {
       scenarioFragment.setScenario(CALL_EXCEPTION);
-      final XCallOobCall oobCall = new XCallOobCall();
-      firstImcFragment.callOob(oobCall);
+      if (opCode == CALL) {
+        final XCallOobCall oobCall = new XCallOobCall();
+        firstImcFragment.callOob(oobCall);
+      }
```

Comment:
Only `CALL` among all CALL-type instructions requires the `OOB_INST_XCALL` OOB instruction.